HowTo/use_gRPC_with_dotnet.md

```diff
@@ -0,0 +1,122 @@
+# How-to deploy a .NET 6 gRPC app on App Service
+
+> [!WARNING]
+> The following documentation is dependent on a future release of App Service that is not currently available
+
+gRPC is a Remote Procedure Call framework that is used to  streamline messages between your client and server over HTTP/2.  Using gRPC protocol over HTTP/2 enables the use of features like multiplexing to send multiple parallel requests over the same connection and bi-directional streaming for sending requests and responses simultaneously.  
+
+gRPC on App Service (Linux only) is now available through the use of a [reverse proxy](https://microsoft.github.io/reverse-proxy/articles/grpc.html) that enables HTTP/2 calls.  The following is a tutorial on how to deploy a .NET 6 gRPC application on App Service. 
+
+#### Prerequisite
+In this tutorial, we'll be deploying a gRPC server to App Service and making a gRPC request to the deployed server from a local gRPC client.  If you have not created a gRPC client and server yet, please follow this [ASP.NET Core tutorial](https://docs.microsoft.com/aspnet/core/tutorials/grpc/grpc-start?view=aspnetcore-6.0&tabs=visual-studio#create-a-grpc-service) to do so.  
+
+The following tutorial builds from the created gRPC client and server in that documentation.  If you already have a gRPC client and server, you may use these steps to add to existing .NET apps as well.
+
+### Setup the gRPC Server app
+In order to prepare our gRPC server application to deploy to App Service, we will need to configure the kestrel to listen to an additional port that only listens for HTTP/2
```

Comment:
"...configure Kestrel to listen to an additional port that only listens for plain-text HTTP/2."